system-addon/test/functional/mochitest/browser.ini

```diff
@@ -1,9 +1,5 @@
 [DEFAULT]
-# XXX This defaults to forcing activity-stream tests to be skipped in m-c,
-# since, as of this writing, mozilla-central itself is still turned off.
-# The tests can be run locally using 'npm run mochitest' which does various
-# overrides.
-skip-if=!activity_stream
+skip-if=!nightly_build
```

Comment:
Actually, I just tested that we don't need to duplicate the logic from `moz.build` and just remove this `skip-if=!nightly_build` line completely. As if activity-stream isn't being built, the test directory won't be processed anyway.